src/app/lib/utilities/usePreviousPath/index.js

```diff
@@ -0,0 +1,11 @@
+import usePrevious from '../usePrevious';
+
+const usePreviousPath = location => {
+  const currentPreviousPath = usePrevious(location.pathname);
+  const oldPreviousPath = usePrevious(currentPreviousPath);
+
+  // set to null on back clicks
+  return location.pathname === oldPreviousPath ? null : currentPreviousPath;
```

Comment:
This will set the `previousPath` to `null` if you go
```
A --click B-> B --click A-> A
```
not just
```
A --click B-> B --navigate back-> A
```